draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.scss

```diff
@@ -5,7 +5,7 @@
 @import "~@kaizen/design-tokens/sass/layout-vars";
 @import "~@kaizen/design-tokens/sass/border-vars";
 @import "~@kaizen/component-library/styles/responsive";
-@import "~@kaizen/component-library/components/Button/styles";
+@import "~@kaizen/draft-button/KaizenDraft/Button/styles";
```

Comment:
These components-depending-on-other-components'-styles give me the willies. I think if anything should be closely checked in these sorts of updates it's this 👁️🔎

Reply:
Yeah, it's not great. There are heaps of places pulling from Button in particular to get the `button-reset` mixin. I'm guessing that's the reason for these, but I'll double check.

Reply:
Yeah as expected it looks like the Sass files I've updated are importing this for `button-reset`, though it's hard to validate 100%. This would be a good thing to add to our lint plugin to make sure this doesn't keep happening.